opm/input/eclipse/EclipseState/Grid/EclipseGrid.cpp

```diff
@@ -1936,6 +1933,13 @@ std::vector<double> EclipseGrid::createDVector(const std::array<int,3>& dims, st
         }
     }
 
+    void EclipseGrid::setMINPVV(const std::vector<double>& minpvv) {
+        if (m_minpvMode == MinpvMode::Inactive) {
+            assert(minpvv.size() == this->getCartesianSize());
```

Comment:
assert on user provided data.

Reply:
Sorry I am too dense. What am I supposed to do here?

Reply:
Maybe it was too early for me. Got it.